src/styles/tablet.css

```diff
@@ -1,3 +1,30 @@
 body {
 	color: blue;
 }
+
+footer .footer {
+	grid-template-columns: 350px 1fr;
+	grid-template-rows: auto;
+	height: 250px;
+}
+
+footer .footer__logo-container {
+	display: flex;
+	align-items: center;
+	width: 80%;
+	margin: 30px auto;
+	padding-right: 30px;
+	border-right: 2px solid #f5f5f5;
+}
+
+footer .footer__links {
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+	justify-content: center;
```

Comment:
I think if you like to center the text you have tu use text-align instead of justify-content, because this line is not doing anything.

Reply:
Removed on next commit